plex/Dockerfile

```diff
@@ -22,7 +25,7 @@ RUN \
     && if [ "${BUILD_ARCH}" = "armv7" ]; then ARCH="armhf"; fi \
     \
     && curl -J -L -o /tmp/plexmediaserver.deb \
-        "https://downloads.plex.tv/plex-media-server-new/1.41.8.9834-071366d65/debian/plexmediaserver_1.41.8.9834-071366d65_${ARCH}.deb" \
+        "https://downloads.plex.tv/plex-media-server-new/${PLEX_VERSION}-${PLEX_HASH}/debian/plexmediaserver_/${PLEX_VERSION}-${PLEX_HASH}_${ARCH}.deb" \
```

Comment:
Might there be a / too much before the second version var?

Reply:
Yup... 🤦
I updated my commit to fix that.